src/components/menu-surface/menu-surface.scss

```diff
@@ -10,7 +10,17 @@
 @include menu.core-styles;
```

Comment:
Please use `menu` as the scope for these commits, not `menu-surface`. `limel-menu-surface` is not a public component.